public/app/features/alerting/unified/utils/amroutes.ts

```diff
@@ -183,7 +183,7 @@ export const formAmRouteToAmRoute = (
     receiver,
   } = formAmRoute;
 
-  const group_by = overrideGrouping && groupBy ? groupBy : [];
+  const group_by = overrideGrouping ? (groupBy ? groupBy : []) : undefined;
```

Comment:
i would add a comment in here, to make more clear we are setting undefined to inherit from parent.
```suggestion
  const inheritFromParent= undefined;
  const group_by = overrideGrouping ? (groupBy ? groupBy : []) : inheritFromParent;
```